core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -499,7 +499,7 @@ object KafkaConfig {
   /** ********* Replication configuration ***********/
   val ControllerSocketTimeoutMsDoc = "The socket timeout for controller-to-broker channels"
   val ControllerMessageQueueSizeDoc = "The buffer size for controller-to-broker-channels"
-  val DefaultReplicationFactorDoc = "default replication factors for automatically created topics"
+  val DefaultReplicationFactorDoc = "default replication factor for automatically created topics, and for AdminClient topic creation requests with -1 in the replication factor"
```

Comment:
Can we add `The` at the start?
I think we can remove AdminClient for the sentence too. Also would the sentence sound better with `as the replication factor` ?

Reply:
@mimaison I have retired and currently don't have an environment set up on my home machines; feel free to take this over and make your suggested changes. Looks like it needs to be reworked anyway due to other changes in these property definitions.

Reply:
No worries, thanks!